Misc/NEWS.d/next/Library/2019-04-01-16-06-36.bpo-31904.peaceF.rst

```diff
@@ -0,0 +1 @@
+Handle VxWorks error string of ENOENT
```

Comment:
Please move this NEWS entry to the Tests category. I suggest:

Fix test_tabnanny on VxWorks: adjust ENOENT error message.